exercise_08/formatStringByWordsLength.js

```diff
@@ -0,0 +1,39 @@
+import getTotalWordsFromString from "../exercise_02/getTotalWordsFromString.js";
+/**
+ * Write JS function "formatStringByWordsLength" to check number of words contained in given string
+    - if total number of words is greater then 5, then return uppercase string
+    - otherwise, return lower case string
+ * input: string (sentence) 
+ * output: string
+ */
+
+/**
+ * Check number of words contained in a given string, and retur uppercase or lowercase string depending on the total number of words
+ * @param {string} sentence
+ * @returns {string} resultSentence
+ */
+
+const formatStringByWordsLength = function (sentence) {
+  let resultSentence = "";
+
+  const numberWords = getTotalWordsFromString(sentence);
+
+  numberWords > 5
```

Comment:
We defined a method to check wether a value is greater or not than a threshold.

Beyond this, you can define this ternary is the proper way:
```js
 resultSentence =  numberWords > 5 // use here abstraction, not implementation
    ? sentence.toUpperCase()
    :  sentence.toLowerCase();
```

FIX: use abstractions, not implementations.